.github/workflows/repo-sync.yml

```
@@ -0,0 +1,28 @@
+name: EdisonEngine-CI
+
+on:
+  schedule:
+    - cron: '0 18 * * *'  # every day at 6 PM : https://crontab.guru/#0_18_*_*_*
+  workflow_dispatch:
+
+jobs:
+  buider:
+    name: buider
+    runs-on: ubuntu-latest
+    steps:
+      - name: "📥 Checkout Code"
+        uses: actions/checkout@v2.3.4
+        with:
+          submodules: "recursive"
+          fetch-depth: 0
+      - name: Get latest CMake and ninja
```

```suggestion
      - name: Install CMake and ninja
```

I'm not sure about this, but I guess the repos in the build image either don't have these tools or they're too outdated. If possible, please install them using `apt-get install`, or add a comment here why these need to be installed using an external action.